dlp/snippets/inspect_content.py

```diff
@@ -983,6 +985,67 @@ def inspect_image_file_all_infotypes(
 # [END dlp_inspect_image_all_infotypes]
 
 
+# [START dlp_inspect_image_listed_infotypes]
+def inspect_image_file_listed_infotypes(
+    project: str,
+    filename: str,
+    info_types: List[str],
+    include_quote=True,
+) -> None:
+    """Uses the Data Loss Prevention API to analyze strings for protected data
+    in image file for given info types.
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        filename: The path to the file to inspect.
+        info_types:
+        include_quote: Boolean for whether to display a quote of the detected
+            information in the results.
+    """
+    # Import the client library
+    import google.cloud.dlp
+
+    # Instantiate a client.
+    dlp = google.cloud.dlp_v2.DlpServiceClient()
+
+    # Prepare info_types by converting the list of strings into a list of
+    # dictionaries (protos are also accepted).
+    info_types = [{"name": info_type} for info_type in info_types]
+
+    # Construct the configuration dictionary. Keys which are None may
+    # optionally be omitted entirely.
+    inspect_config = {
+        "info_types": info_types,
+        "include_quote": include_quote,
+    }
+
+    # Construct the byte_item, containing the image file's byte data.
+    with open(filename, mode="rb") as f:
+        byte_item = {"type_": 'IMAGE', "data": f.read()}
+
+    # Convert the project id into a full resource id.
+    parent = f"projects/{project}"
+
+    # Call the API.
+    response = dlp.inspect_content(
+        request={
+            "parent": parent,
+            "inspect_config": inspect_config,
+            "item": {"byte_item": byte_item},
+        }
+    )
+
+    # Print out the results.
+    if response.result.findings:
```

Comment:
Better result formatting here:

- Add an empty line between results to differentiate them a bit more
- Consider moving InfoType to be first since that's the most important part.
- Also consider hiding the "Quote" line unless include_quote=True, otherwise it will be empty in the response.

Reply:
updated, Thank you for reviewing